### PR TITLE
Refactoring environment variables for easier k8s deployment

### PR DESCRIPTION
**Pull request by slowr**
### Description of PR

What component(s) does this PR affect?

- [ ] Back-End (Database, Microservices, Containers, etc)
- [ ] Front-End (Flask, API, etc)
- [ ] Docs
- [x] Build System

Does the PR require changes on other components? If yes, please mark the components:

- [ ] Back-End (Database, Microservices, Containers, etc)
- [ ] Front-End (Flask, API, etc)
- [ ] Docs
- [ ] Build System

### Related Issue

Resolves #

### Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Docs update
- [ ] None of the above

### Checklist:

- [x] I have read the **[contributing guide](https://github.com/FORTH-ICS-INSPIRE/artemis/blob/master/CONTRIBUTING.md)** and my code conforms to the guidelines.
- [ ] This change requires a change in the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -4,17 +4,20 @@ COMPOSE_PROJECT_NAME=artemis
 DB_VERSION=12
 GUI_ENABLED=true
 SYSTEM_VERSION=latest
+HISTORIC=false
```
Comment (vkotronis) [inline .env]: make sure you document all the updated .env variables in the respective wiki page

File: backend/entrypoint [hunk 2]
```diff
@@ -1,17 +1,16 @@
...
     fi
-    service cron start
```
Comment (vkotronis) [inline backend/entrypoint]: this is redundant right?

File: backend/entrypoint.test [hunk 3]
```diff
@@ -1,16 +1,15 @@
 #!/bin/bash
-./wait-for rabbitmq:5672 -t 0
-./wait-for postgres:5432 -t 0
+./wait-for ${RABBITMQ_HOST}:${RABBITMQ_PORT} -t 0
+./wait-for ${DATABASE_HOST}:${DATABASE_PORT} -t 0
 
-version=$(PGPASSWORD=$DATABASE_PASSWORD psql -U $DATABASE_USER -d $DATABASE_NAME -h $DATABASE_HOST -c "select version from db_details;" -tA)
+version=$(PGPASSWORD=$DATABASE_PASS psql postgres://${DATABASE_USER}:${DATABASE_PASS}@${DATABASE_HOST}:${DATABASE_PORT}/${DATABASE_NAME} -c "select version from db_details;" -tA)
```
Comment (vkotronis) [inline backend/entrypoint.test]: this is fully equivalent to the new version? Just make sure to test.

File: backend/migrate/migrate.py [hunk 4]
```diff
@@ -45,10 +45,11 @@ def create_connect_db():
             _db_name = os.getenv("DATABASE_NAME", "artemis_db")
```
Comment (vkotronis) [inline backend/migrate/migrate.py]: make sure you test the migration before merging (e.g., from version 11 to 12, or simply a custom version)

File: docker-compose.yaml [hunk 5]
```diff
@@ -102,9 +112,8 @@ services:
         networks:
             - artemis
         expose:
-            - 5671
```
Comment (vkotronis) [inline docker-compose.yaml]: why were these exposed before?

Comment (slowr) [reply docker-compose.yaml]: they were never needed

File: backend/entrypoint.test (hunk 3 above)
Comment (slowr) [reply backend/entrypoint.test]: it's the same command

File: backend/entrypoint (hunk 2 above)
Comment (slowr) [reply backend/entrypoint]: isn't used at all

File: backend/migrate/migrate.py (hunk 4 above)
Comment (slowr) [reply backend/migrate/migrate.py]: it's the same command

**Comment by slowr** [comment]
@vkotronis I tested this PR and didn't find any issue. kubernetes is delete because it is handled on the other PR.

**Pull request merged**